src/core/column-type/column.model.js

```diff
@@ -10,7 +10,7 @@ export class ColumnModel {
 		this.pin = null;
 		this.origin = 'specific';
 		this.source = 'user';
-		this.class = 'data';
+		this.class = '';
```

Comment:
I think when user create own column he shouldn't think about it class, class is done for internal uses to the most of time